src/assets/css/dateTimeRangePickerDropdown.css

```diff
@@ -0,0 +1,4 @@
+/* CSS to fix the daterange dropdown picker issue */
+.md-drppicker.drops-down-auto.ltr.shown.double.show-ranges {
```

Comment:
Could be put in src/assets/scss/views/shared/_date-picker.scss ? or a new file like _date_time_range_picker.scss?